resources/benchmark-runner.mjs

```diff
@@ -41,43 +41,71 @@ class BenchmarkState {
         return !this._testIndex;
     }
 
-    prepareCurrentSuite(runner, frame) {
+    async prepareCurrentSuite(frame, page) {
         const suite = this.currentSuite();
-        return new Promise((resolve) => {
-            frame.onload = () => {
-                suite.prepare(runner, frame.contentWindow, frame.contentDocument).then(resolve);
+        await new Promise((resolve) => {
+            frame.onload = async () => {
+                await suite.prepare(page);
+                resolve();
             }
             frame.src = 'resources/' + suite.url;
         });
     }
 }
 
-export class BenchmarkRunner {
-    constructor(suites, client) {
-        this._suites = suites;
-        this._prepareReturnValue = null;
-        this._client = client;
+const ENTER_KEY_CODE = 13;
```

Comment:
It doesn't seem like this needs to be global?
We can just declare it within `triggerEnter` instead.